src/bodypart.cpp

```diff
@@ -349,7 +349,14 @@ void body_part_type::load( const JsonObject &jo, const std::string & )
         limbtypes.clear();
         body_part_type::type limb_type;
```

Comment:
Why not add a default value here
```suggestion
        body_part_type::type limb_type = {};
```
pragma diagnostic is the last resort

Reply:
Happy to go that route. Thanks for the review.